docs/source/user_guide/installation/install-from-docker.rst

```diff
@@ -85,6 +85,9 @@ Once you have pulled the Docker image, you can run a Docker container with the P
 
 3. You can execute PyBaMM-related commands, run tests develop & contribute from the container.
 
+.. note::
+    The default user for the container is ``pybamm`` with ``pybamm`` as password. The user belongs to ``sudoers`` and ``root`` group so the sudo command can be issued to install additional packages to the container.  After a clean install, ``sudo apt-get update`` should be executed to update the source list. Additional packages can be installed using ``sudo apt-get install [package_name]``.
```

Comment:
```suggestion
.. note::

    The default user for the container is ``pybamm`` with ``pybamm`` as password. The user belongs to
    ``sudoers`` and ``root`` group, so the sudo command can be issued to install additional packages to
    the container.  After a clean install, ``sudo apt-get update`` should be executed to update the source
    list. Additional packages can be installed using ``sudo apt-get install [package_name]``.
```

Just a formatting change, please split text into multiple lines if possible and indent the text if you are using an admonition